storage/innobase/xtrabackup/doc/source/advanced/encrypted_innodb_tablespace_backups.rst

```diff
@@ -353,6 +355,22 @@ The backup is now prepared and can be restored with :option:`--copy-back` option
 case the keyring has been rotated you'll need to restore the keyring which was
 used to take and prepare the backup.
 
+Using Key Management Interoperability Protocol
+==============================================
+
+This feature is *tech preview* quality.
+
+Percona XtraBackup 8.0.27-19 adds support for the Key Management Interoperability Protocol (KMIP) which enables the communication between the key management system and encrypted database server.
+
+Percona XtraBackup has no special requirements for backing up a database that contains encrypted InnoDB tablespaces. 
+
+Percona XtraBackup performs the following actions:
+
+1. Connects to the MySQL server
+2. Pulls the configuration
+3. Connects to the KMIP server
+4. Fetches the necessary keys from the KMIP server.
```

Comment:
Are these keys stored in the backup directory? Good to have this info

Reply:
Updated